### PR TITLE
 finsihed faq questions required ui implementation

### PR DESCRIPTION
**Pull request by rohan2734**

### CODE REVIEW

**Review by ExpressHermes — APPROVED** [review]

**Review by mann27 — APPROVED** [review]

**Comment by mann27** [comment]
give @rohan2734 75 points

**Comment by rohan2734** [comment]
@mann27 @ExpressHermes  there is one small wierd problem, that is the plus signs are initially visible for the first two faq, and not the other two, but when u click the other two on the question, then the plus and minus signs show normal behaviour.

but i dont know why it is happening like this, is it due to margintop? on the question?

**Comment by rohan2734** [comment]
and can i add some animation?

File: src/components/HelpPage/Faqs/index.js [hunk 1]
```diff
@@ -1,36 +1,89 @@
...
+    {
+        id:2,
+        question:"What should I do if I find any post abusive or against someone?",
+        answer:"You can report the post and it will be taken into consideration by the admin if it is necessary to report the post.",
+        isShow:false
+    },
+    {
+        id:3,
+        question:"Can I post something which is articulated by me?",
+        answer:"No, you just have to post the link of an article with its title, any relevant image and its corresponding tags."
```
Comment (ExpressHermes) [inline src/components/HelpPage/Faqs/index.js:21]: @rohan2734 You have added `isShow` in first two questions only. Add that in all the questions.

**Review by ExpressHermes — CHANGES_REQUESTED** [review]

File: src/components/HelpPage/Faqs/index.js [hunk 2]
```diff
@@ -1,36 +1,89 @@
...
+    },
```
Comment (mann27) [inline src/components/HelpPage/Faqs/index.js:22]: isShow here is not set to false by default in id:3,4

**Comment by mann27** [comment]
No, we don't need animation here. Also, get rid of the image in the page.

**Review by mann27 — COMMENTED** [review]
Resolve all the compile warnings in the `./src/components/HelpPage/Faqs/index.js
`

**Comment by mann27** [comment]
The revert was a mistake and I haven't reverted anything. As I have merged these changes simply make a new PR and that will be considered into this only.

**Pull request merged**